product_brand/__manifest__.py

```diff
@@ -8,18 +8,20 @@
 
 {
     'name': 'Product Brand Manager',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Product',
     'summary': "Product Brand Manager",
     'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
               ', MONK Software, Odoo Community Association (OCA)'
               ', SerpentCS Pvt. Ltd.',
     'license': 'AGPL-3',
-    'depends': ['product'],
+    'depends': [
+        'sale',
```

Comment:
He uses it for the menu positioning, he uses `sale.prod_config_main` as parent menu.
I'm not sure if there is a good lower-level menu to use as reference for positioning